src/components/common/CopyToClipboard.tsx

```diff
@@ -0,0 +1,43 @@
+import { makeStyles, Tooltip } from '@material-ui/core';
+import { useSnackbar } from 'notistack';
+import React from 'react';
+
+const useStyles = makeStyles(() => ({
+  withPointer: {
+    cursor: 'pointer',
+  },
+  container: {
+    display: 'inline-block',
+  },
+}));
+
+interface CopyToClipboardProps {
+  text: string;
+  children: React.ReactNode;
+  successMessage?: string;
+}
+const CopyToClipboard = (props: CopyToClipboardProps) => {
+  const { successMessage, children, text } = props;
+  const { enqueueSnackbar } = useSnackbar();
+  const classes = useStyles();
+
+  const handleClick = () => {
+    enqueueSnackbar(successMessage ?? 'Copied to clipboard', {
+      variant: 'success',
+    });
+    navigator.clipboard.writeText(text);
+  };
+
+  return (
+    <Tooltip
+      onClick={handleClick}
+      color="primary"
+      title="Copy to clipboard"
+      className={classes.withPointer}
+    >
+      <div className={classes.container}>{children}</div>
```

Comment:
I think in cases like this one it is much better to use something like `<Box display="inline-block">{children}</Box>` instead of using div and defining custom class just for the display property. I think we should start using the `Box` component more because it accepts many customizations.

Reply:
I have changed this component a little and not using ToolTip
![image](https://user-images.githubusercontent.com/58165815/160563681-498ac63e-7b34-428f-b322-02f86be7a5d8.png)
I think too that we could use Box more as it has a lot of CSS as attributes, so we can reduce amount of makeStyles